From fcce19d5af891c44f54ee96f932f7b3457b8a6d3 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 16 May 2017 23:57:00 +0800
Subject: [PATCH 1794/5242] MLK-14933 gpio: kconfig: remove mxc gpio
 dependency

commit  4f665449323d201445202d0d8290ae2d30cd4ae3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove dependency of ARCH_MXC to support GPIO driver on
both ARMv7 and ARMv8 i.MX platforms.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpio/Kconfig |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index 71c0ab4..af37270 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -368,7 +368,6 @@ config GPIO_MVEBU
 
 config GPIO_MXC
 	def_bool y
-	depends on ARCH_MXC
 	select GPIO_GENERIC
 	select GENERIC_IRQ_CHIP
 
-- 
1.7.9.5

